### DISCUSSION TITLE
Interested in contributing

### DISCUSSION THREAD

**Discussion by Ekaums** [bpg/terraform-provider-proxmox#1461] [General]
Hi, I have been using this plugin for a bit and have loved it. I was interested in contributing but wasn't sure where to start, any good first issues/tasks to pick up? I have a bit of experience in Go but not too much, but I am an intermediate programmer and I would love to learn more :)

**Comment by bpg-dev** [comment]
Hi @Ekaums 👋🏼 

That's great! I don't have many easy tickets at the moment, but I guess documentation can always be improved. To start, you could try setting up your environment to be able to run make example using API Token authentication and a non-root user, following the documentation. I'm pretty sure you'll encounter some unclear moments or other deficiencies, and you'll be in a good position to contribute improvements!
